package.json

```diff
@@ -2,8 +2,9 @@
   "name": "edx",
   "version": "0.1.0",
   "dependencies": {
+    "backbone": "1.3.3",
```

Comment:
Don't put backbone here yet! We are not ready to install this via NPM (the Lahore team is working on a Backbone upgrade).